src/templates/staking.tsx

```diff
@@ -344,11 +344,12 @@ const StakingPage = ({
   data: { pageData: mdx },
   location,
 }: PageProps<Queries.StakingPageQuery, Context>) => {
-  if (!mdx?.frontmatter) {
+  if (!mdx?.frontmatter || !mdx?.frontmatter?.title)
     throw new Error(
       "Staking page template query does not return expected values"
     )
-  }
+  if (!mdx?.frontmatter?.title)
+    throw new Error("Required `title` property missing for staking template")
```

Comment:
@pettinarip Do you want to add this `OR` statement here? Just noting as-is, this second `if` statement will never run

ie: If (!mdx?.frontmatter?.title) => Throws the first error, and will never get to the second

Reply:
Ouch 🤦🏼 fixing!